From 980c9de9e3a117357e037032f32c740bcb6bd9b9 Mon Sep 17 00:00:00 2001
From: Michael Roth <mdroth@linux.vnet.ibm.com>
Date: Mon, 5 Mar 2012 17:44:01 -0500
Subject: [PATCH 34/98] qerror: add QERR_JSON_PARSE_ERROR to qerror.c

Missing from previous addition of error to qerror.h. Needed for
qerror_format() and friends.

Signed-off-by: Michael Roth <mdroth@linux.vnet.ibm.com>
Signed-off-by: Luiz Capitulino <lcapitulino@gmail.com>
(cherry picked from commit ac32c7807640f04682ea17bca17c22f8b9264d62)
Signed-off-by: Jeff Cody <jcody@redhat.com>
Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 qerror.c |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/qerror.c b/qerror.c
index 5f0668d..e76f487 100644
--- a/qerror.c
+++ b/qerror.c
@@ -145,6 +145,11 @@ static const QErrorStringTable qerror_table[] = {
         .desc      = "Invalid JSON syntax",
     },
     {
+        .error_fmt = QERR_JSON_PARSE_ERROR,
+        .desc      = "JSON parse error, %(message)",
+
+    },
+    {
         .error_fmt = QERR_KVM_MISSING_CAP,
         .desc      = "Using KVM without %(capability), %(feature) unavailable",
     },
-- 
1.7.7.6

